stdlib-integration-tests/index.json

```diff
@@ -2,5 +2,9 @@
   {
     "name": "Socket standard library ",
     "path": "socket"
+  },
+  {
+    "name": "Socket standard library ",
```

Comment:
Shouldn't we maintain seperate name for task rather than using socket standard library?